network-api/networkapi/templates/fragments/buyersguide/in_the_press.html

```diff
@@ -0,0 +1,26 @@
+{% load static wagtailcore_tags wagtailimages_tags i18n %}
+<div>
+    <img src="{% static "_images/buyers-guide/long-wavy-border.svg" %}" class="tw-w-full" />
+    <h2 class="tw-font-zilla tw-font-normal tw-text-2xl tw-text-black tw-mt-2 tw-mb-[20px]">{% trans "In the press" %}</h2>
+
+    <ul class="tw-list-none tw-space-y-4 tw-p-0">
+        {% for update in product_updates %}
+            <li class="tw-m-0 tw-min-h-[78px] tw-flex tw-flex-col tw-justify-center" >
```

Comment:
```suggestion
            <li class="tw-m-0 tw-flex tw-flex-col tw-justify-center" >
```

Hey Simon! Suggesting this because the list items that have single line titles look a little more spaced out than the figma rendition. I think with real data, titles are typically going to have two lines like the figma comp, and will look spaced out as expected 👍